plenum/server/config_req_handler.py

```diff
@@ -52,23 +53,56 @@ def updateState(self, txns, isCommitted=False):
 
     def updateStateWithSingleTxn(self, txn, isCommitted=False):
         typ = get_type(txn)
-        payload = get_payload_data(txn)
         if typ == TXN_AUTHOR_AGREEMENT:
-            self.update_txn_author_agreement(payload)
-
-    def update_txn_author_agreement(self, payload):
-        version = payload[TXN_AUTHOR_AGREEMENT_VERSION]
-        text = payload[TXN_AUTHOR_AGREEMENT_TEXT]
+            payload = get_payload_data(txn)
+            self.update_txn_author_agreement(
+                payload[TXN_AUTHOR_AGREEMENT_VERSION],
+                payload[TXN_AUTHOR_AGREEMENT_TEXT],
+                get_seq_no(txn),
+                get_txn_time(txn)
+            )
+
+    def update_txn_author_agreement(self, version, text, seqNo, txnTime):
```

Comment:
In a new code we use snake_case as much as we can, so probably `seq_no` and `txn_time` would be better parameter names.

Reply:
fixed